### ISSUE TITLE
Use sasslintConfig var in package.json like sass-lint 

### ISSUE THREAD

**Issue by refactorized** [sasstools/sass-lint#1039]
Please add ability to specify config in any upstream package.json file.  sass-lint uses the `sasslintConfig` key.  

This could be considered a partial dupe of https://github.com/sasstools/sass-lint/issues/645 "Use cosmiconfig for configuration discovery" which should address it implicitly.  However, if cosmiconfig is _not_ used I think package specified configs are still important to support.

**Comment by DanPurdy** [comment]
I agree, definitely and had something in the works before I dropped off the face of the planet for a while. Will look to resurrect it now and get something together for it.

**Comment by OlehDutchenko** [comment]
I needed it!
